src/pages/selection/candidates/index.jsx

```diff
@@ -1,76 +1,105 @@
       }
-    }
-    );
+    });
   }
   componentWillUnmount() {
     this._isMounted = false;
   }
   loadCandidatesApi = async () => {
-    const response = await pyv.get('/api/candidates');
-    const response2 = await pyv.get('/api/races'); 
-    return [response.data, response2.data];
+    // const response = await pyv.get('/api/candidates');
+    const response = await pyv.get('/api/races');
+    return response.data;
+  };
+
+  selectBtn = data => {
+    let temp = {};
+    temp['candidateId'] = data.candidateId;
+    temp['name'] = data.name;
+    temp['electionId'] = data.electionId;
+    temp['election'] = data.election;
+    temp['details'] = data.details;
+    temp['organizationId'] = data.organizationId;
+    temp['organization'] = data.organization;
+    temp['candidateRaces'] = data.candidateRaces;
+    temp['contacts'] = data.contacts;
+
+    this.state.selectedCandidates.push(temp);
+
+    localStorage.setItem(
+      'selectedCandidateRaces',
+      JSON.stringify(this.state.selectedCandidates)
+    );
   };
 
   render() {
-    const {candidatesHeader} = this.state
+    const { candidatesHeader } = this.state;
     const cardStyle = {
       maxWidth: '540px'
     };
 
-
-    let candidates = this.state.candidates.map(cData => {
-      return (
-        <div className='col-sm-3' key={cData.candidateId}>
+    let candidates = this.state.races.map(rData => {
+      return rData.candidates.map(cData => {
+        return (
+        <div className='col-sm-3' key={cData.candidate.candidateId}>
           <div className='card' style={cardStyle}>
-            <img src={`${IMAGE_BASE}/${cData.picture}`} className='card-img-top' alt='...' />
+            <img
+              src={`${IMAGE_BASE}/${cData.candidate.picture}`}
+              className='card-img-top'
+              alt='...'
+            />
             <div className='card-body'>
-              <h5 className='card-title'>{cData.name}</h5>
+              <h5 className='card-title'>{cData.candidate.name}</h5>
               {/* <h6 className="card-subtitle mb-2 text-muted">cData.organization</h6> */}
               {/* <h6 className="card-subtitle mb-2 text-muted">cData.position</h6> */}
               {/* <p className='card-text'>
                 Some quick example text to build on the card title and make up
                 the bulk of the card's content.
               </p> */}
-              <a href='#' className='btn btn-primary'>
+              <button
+                className='btn btn-primary'
+                onClick={e => this.selectBtn(cData.candidate)}
+              >
                 Select
-              </a>
+              </button>
             </div>
           </div>
         </div>
       );
-    });
+      })
+    })
 
     return (
       <div className='container'>
         <div className='row'>
           <SectionHeader
-            title={candidatesHeader.pageTitle} 
+            title={candidatesHeader.pageTitle}
             subtitle=''
```

Comment:
We don't need to put `subtitle=''` when we don't have any subtitle. We can remove this line.

Reply:
Fixed!